metamorph/metamorph_plugin.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import logging.config
+import json
+import time
+import os
+
+import requests
+
+
+class MetamorphPlugin(object):
+
+    MINUTE = 60
```

Comment:
I truly think it's not needed to define a constant called MINUTE. It would make little since to assign it any other value that 60, so why not just use sixty where you're calling self.MINUTE?